docs/reference/line-endings.md

```
@@ -0,0 +1,75 @@
+# Why every file in this repo is LF
+
+## The short version
+
+`.gitattributes` starts with one line:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)